.tools/psalm/baseline-taint.xml

```diff
@@ -53,11 +53,6 @@
       <code>rex_mediapool_Uploadform($rex_file_category)</code>
     </TaintedHtml>
   </file>
-  <file src="redaxo/src/addons/structure/plugins/content/lib/article_content_base.php">
-    <TaintedHtml occurrences="1">
-      <code>$articleContent</code>
-    </TaintedHtml>
-  </file>
```

Comment:
Ich kapiere nicht wirklich, weshalb der Eintrag jetzt wegfällt.

Reply:
Kann man irgendwie sehen welche zeile das genau in der datei ist?

Reply:
Wüsste nicht wie.

Reply:
Ggf. Psalm im aktuellen master ohne diese zeile aus der baseline laufen lassen und schauen welche zeile angemeckert wird?

Reply:
das hatte ich gemacht, und hatte dann plötzlich ganz viele errors, was ich nicht verstanden habe, und wo ich dann auch nicht durchgestiegen bin.

ich bin sowieso dran, für rex_type die taint analyse zu optimieren, benötige da aber noch einen psalm fix, der schon da ist, aber noch nicht released.
ich denke in dem rahmen schaue ich mir dann auch rex_sql nochmal genauer an mit den neuen types hier.